### PR TITLE
Observability

### PR DESCRIPTION
**Pull request by Joshocan**

### CODE REVIEW

File: technical-infrastructure/observability/availability.md [hunk 1]
```diff
@@ -8,21 +8,22 @@ nav_order: 3951
...
 
 The status of a service is typically indicated as up/down or available/unavailable.
 
 ## Availability and Status Monitoring
 
-All CESSDA services are monitored by [Uptime Robot](https://uptimerobot.com/).
-Each service endpoint is sent a ping request every 5 minutes.
+All CESSDA service endpoints are monitored by
+[Uptime Robot](https://uptimerobot.com/).
+A ping rrquest from Uptime Robot is sent to each endpoints every 5 minutes.
```
Comment (john-shepherdson) [inline technical-infrastructure/observability/availability.md:23]: Typo on ´request´

File: technical-infrastructure/observability/index.md [hunk 2]
```diff
@@ -25,19 +22,19 @@ The observability strategy is made up of three pillars:
 1. **Metrics Monitoring:** Metrics represent the raw measurements of resource usage or behaviour that can be
   observed and collected throughout the infrastructure.
   The [monitoring system]({% link technical-infrastructure/observability/monitoring-overview.md %}) collects, exposes,
-  and analyses those values to improve awareness of the characteristics and behaviour of the components
-  the the core products are composed of.
+  and analyses those values to improve the services and optimise the infrastructure as a whole.
 1. **Logging & aggregation:** Logs are detailed records of events that occur within the components
   or the Infrastructure in general.
-  The [central logging system]({% link technical-infrastructure/observability/logging-overview.md %}) collects, aggregates,
+  CESSDA has [central logging system]({% link technical-infrastructure/observability/logging-overview.md %}) that collects, aggregates,
```
Comment (john-shepherdson) [inline technical-infrastructure/observability/index.md:28]: Typo ´CESSDA has a´

File: technical-infrastructure/observability/index.md [hunk 3]
```diff
@@ -25,19 +22,19 @@ The observability strategy is made up of three pillars:
...
   filters and enriches the contents of the logs and enables visualisation of system and product events,
-  which can aid debugging and problem solving.
+  which then aid debugging process and problem solving capabilities.
```
Comment (john-shepherdson) [inline technical-infrastructure/observability/index.md:30]: typo ´aid the´

File: technical-infrastructure/observability/monitoring-overview.md [hunk 4]
```diff
@@ -10,88 +10,101 @@ nav_order: 3953
...
 
 - **D**uration - distributions of the amount of time each request takes
 
-## Metrics exposure
+## Metrics Instrumentation and Exposure
 
-Infrastructure components and services are instrumented within the Prometheus stack, depending on
-whether they are customised applications or off-the-shelf tools.
+Infrastructure components and services utilise Prometheus libraries for
+instrumentation, tailoring the intrgration to suit the requirements of
```
Comment (john-shepherdson) [inline technical-infrastructure/observability/monitoring-overview.md:52]: Typo on ´integration´

File: technical-infrastructure/observability/monitoring-overview.md [hunk 5]
```diff
@@ -10,88 +10,101 @@ nav_order: 3953
...
 
-- The Prometheus server consist of a time series database (tsdb), a data retrieval agent, and a web server.
- The data retrieval agent is used to discover targets in Kubernetes and pull metrics from exporters or applications
- exposed with clients libraries and stored in a time series database.
- PromQL is a functional query language that is used to select and aggregate time-series data from the
- Prometheus database.
+The Prometheus server consist of a time series database (tsdb),
+a data retrieval agent, and a web server. The data retrieval agent is used to
+discover targets in Kubernetes and pull metrics from exporters or applications
+exposed with clients libraries and stored in a time series database. PromQL is
```
Comment (john-shepherdson) [inline technical-infrastructure/observability/monitoring-overview.md:77]: clients -> client
stored -> store them

File: technical-infrastructure/observability/monitoring-overview.md [hunk 6]
```diff
@@ -10,88 +10,101 @@ nav_order: 3953
...
 
-- Alertmanger and Grafana are only deployed on the Management cluster to centrally send alert notifications and
-  visualize and analyze metrics respectively.
+There are other exporters deployed for third party tools like Jenkins,
+Elasticsearch and HaProxy, depending on which tools are running in the cluster.
 
-- Metrics data stored in a time series database are retained on persistent volumes in the clusters.
-  The retention policy set on Prometheus is based on ensuring that the data does not exceed the available storage space.
+ Grafana is only deployed on the Management cluster to centrally send alert
+notifications and visualize and analyze metrics respectively.
```
Comment (john-shepherdson) [inline technical-infrastructure/observability/monitoring-overview.md:101]: visualize -> visualise
analyze -> analyse

File: technical-infrastructure/observability/monitoring-overview.md [hunk 7]
```diff
@@ -10,88 +10,101 @@ nav_order: 3953
...
+
+Alertmanager is disabled in all the environments excepts for Management and
+productuon clusters.
+
+ Metrics data stored in a time series database are retained on persistent
```
Comment (john-shepherdson) [inline technical-infrastructure/observability/monitoring-overview.md:106]: data is stored

**Review by john-shepherdson — COMMENTED** [review]
Please fix typos

File: technical-infrastructure/observability/availability.md [hunk 8]
```diff
@@ -8,21 +8,22 @@ nav_order: 3951
...
 
-All CESSDA services are monitored by [Uptime Robot](https://uptimerobot.com/).
-Each service endpoint is sent a ping request every 5 minutes.
+All CESSDA service endpoints are monitored by
+[Uptime Robot](https://uptimerobot.com/).
+A ping request from Uptime Robot is sent to each endpoints every 5 minutes.
 If a timely response is not received, Uptime Robot sends an alert message,
 which is picked up by CESSDA's
-[Observability framework]({% link technical-infrastructure/observability/index.md %})
+[Helpdek system](https://helpdesk.cessda.eu/#dashboard)
```
Comment (alencessda) [inline technical-infrastructure/observability/availability.md:26]: Typo: Helpdesk

**Review by alencessda — CHANGES_REQUESTED** [review]
Please check the comments on files. There are still a few typos and an update based on recent UptimeRobot subscription.

**Comment by matthew-morris-cessda** [comment]
This is unlikely to be merged now

**Pull request closed without merging**